app/views/accountants/_table_content.html.erb

```diff
@@ -0,0 +1,33 @@
+<!-- locals: -->
+
+<div id="<%= table_type %>" class="margin-bottom">
+  <h1 class="border-bottom title"><%= title %></h1>
+  <table class="table border-side">
+    <thead>
+      <tr>
+        <% if local_assigns[:sortable] %>
```

Comment:
Go ahead and pop this out -- it's a holdover from the call list builder. No need.